### Excepciones ###

# Manejo de errores - Exception Handling

numberOne = 1
numberTwo = 5
numberTwo = "1"

print(numberOne + numberTwo)